crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj

```diff
@@ -138,6 +145,7 @@
 				1073482720333B71004A66D1 /* CrashlyticsExampleUITests */,
 				5F5A534D1ADE670C00F81DF0 /* Products */,
 				5F9961041AE0CF4F0034F503 /* Shared */,
+				3E68189DB1DD7B4868EE2558 /* Pods */,
```

Comment:
And here.

Deleting the red Pods folders in Xcode is another way to manage this.

Reply:
Done.